lib/process/process.go

```diff
@@ -1779,6 +1812,17 @@ func (p *Process) initAccount() error {
 	return nil
 }
 
+// removeLegacyIdentities removes legacy admin/proxy identities so that new
+// ones can be generated new ones upon the first start of new teleport
```

Comment:
@a-palchikov Yes, it won't start, it'll fail on some internal teleport migration cause 3.0 is not fully backward-compatible with 2.4 (we skipped 4 releases). I've added this to the comment.

@knisbet It will regenerate identities in the old format just like on fresh install.